lib/Backends/NNPI/tests/NNPIOperatorTest.cpp

```diff
@@ -95,12 +95,10 @@ struct BlacklistInitializer {
             {"LayerNorm_Float/0", TestBlacklist::AnyDeviceHWEngine},
             {"LengthsSum/0", TestBlacklist::AnyDeviceAnyEngine},
             {"LengthsToRanges/0", TestBlacklist::AnyDeviceAnyEngine},
-            {"ModuloInt32NoSignFollow/0", TestBlacklist::AnyDeviceAnyEngine},
-            {"ModuloInt32SignFollow/0", TestBlacklist::AnyDeviceAnyEngine},
             {"ModuloInt64NoSignFollow/0", TestBlacklist::AnyDeviceAnyEngine},
             {"ModuloInt64SignFollow/0", TestBlacklist::AnyDeviceAnyEngine},
-            {"mul_int32/0", TestBlacklist::AnyDeviceHWEngine},
-            {"mul_int64/0", TestBlacklist::AnyDeviceHWEngine},
+            {"mul_int32/0", TestBlacklist::AnyDeviceAnyEngine},
+            {"mul_int64/0", TestBlacklist::AnyDeviceAnyEngine},
```

Comment:
Is this a temporary regression?

Reply:
These tests where failing on HW and we updated the isOpSupported to return false.

Reply:
Ok, but it used to work on ICEREF. Are these going to be fixed and re-enabled for the next release, on ICEREF and on HW?

Reply:
There are no plans to support it in HW, it was removed to align with HW. 
The current code without changes returns a wrong isOpSupported value for HW.